setup.cfg

```diff
@@ -41,6 +41,7 @@ keywords =
 packages = find:
 install_requires =
     Django>=1.8,<5
+    django-ipware>=4
```

Comment:
**Question:** Do we worry about any incompatibilities later down the line or we trust the library users? :D I think I'm fine with the lax requirement, given it's usually more pain to release...